### DISCUSSION TITLE
Feature Request: Remove followers

### DISCUSSION THREAD

**Discussion by cregganna** [bluesky-social/atproto#3180] [General Discussion]
Would it be possible to either stop people following me or have a way of removing followers. The only option I have is to block which I only want to do for spam, bots, and unsavoury people 

Thx for a great service.

**Comment by 5brian** [comment]
See: https://github.com/bluesky-social/social-app/issues/1160#issuecomment-1677642129

**Comment by cregganna** [comment]
This is duplicate of https://github.com/bluesky-social/social-app/issues/1160
